apps/payments/next/app/[locale]/[offeringId]/checkout/[interval]/[cartId]/start/page.tsx

```diff
@@ -19,12 +24,28 @@ export default async function Checkout() {
   const locale = headers().get('accept-language') || DEFAULT_LOCALE;
   const sessionPromise = auth();
   const l10nPromise = app.getL10n(locale);
-  const [session, l10n] = await Promise.all([sessionPromise, l10nPromise]);
+  const cartPromise = getCartAction(params.cartId);
+  const [session, l10n, cart] = await Promise.all([
+    sessionPromise,
+    l10nPromise,
+    cartPromise,
+  ]);
+
+  // Temporary function used to test handleStripeErrorAction
+  // This is to be deleted as part of FXA-8850
+  async function onSubmit() {
+    'use server';
+    const error: Stripe.StripeRawError = new Error(
+      'Whoops'
```

Comment:
Nit: While this is only temporary a temporary placeholder, would consider making the text a bit more descriptive to keep it clean (in case for whatever reason that work gets delayed).